js/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "inji-vci-client",
+  "version": "1.0.0",
+  "description": "Contains OpenId4VCI specification related source code and documentation",
```

Comment:
Write proper one-liner description about this library as it will be visible on npm.